### PR TITLE
New: [AEA-4223] - PSU Automated Testing Happy Path

### PR DESCRIPTION
**Pull request by seansteberisal**
## Summary

- :sparkles: New Feature

### CODE REVIEW

File: features/steps/psu_api_steps.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+@given("I am authorised to send prescription updates")
+def i_am_authorised_to_send_prescription_updates(context):
+    env = context.config.userdata["env"].lower()
+    if "sandbox" in env:
+        return
+    context.auth_token = get_auth(env, "PSU")
+
+
+@when("I send an update")
```
Comment (anthony-nhs) [inline features/steps/psu_api_steps.py:19]: should we name this 'I send a collected update' (or similar) so we can have other steps in future to send different updates

File: package.json [hunk 2]
```diff
@@ -1,11 +1,12 @@
 {
   "devDependencies": {
-    "@semantic-release/changelog": "^6.0.3",
-    "@semantic-release/release-notes-generator": "^14.0.1",
+    "@semantic-release/changelog": "^5.0.1",
```
Comment (anthony-nhs) [inline package.json:3]: why are you downgrading versions?

Comment (seansteberisal) [reply package.json:3]: Not sure how this happened, could be due to pyright problems i was facing, reverted

File: features/steps/psu_api_steps.py (hunk 1 above)
Comment (seansteberisal) [reply features/steps/psu_api_steps.py]: Turned this into a parameterised step so we can pass in the status and terminal status each time we want to run this step

**Review by anthony-nhs — APPROVED** [review]

**Pull request merged**